### ISSUE TITLE
Panel keeps redirecting on clean install

### ISSUE THREAD

**Issue by danoszz** [brocessing/kirby-webpack#38]
When having a clean install of 'kirby-webpack' (+ [onboil](https://github.com/brocessing/unboil) 👀) on Mac OSX the /panel keeps redirecting back to the current page. No errors are shown except the warning:

```
⚠  Warning: 
Node can't reach PHP built-in server through localhost.
Proxying [::1]:35411 instead.
```
This warning was explained in [this closed issue](https://github.com/brocessing/kirby-webpack/issues/37), so that isn't the problem I guess.

**Comment by grommas** [comment]
have the same issue since today with a forked version (for scss)

**Comment by pqml** [comment]
I tried it, there is indeed an issue, maybe something with `node-proxy`

A quick and dirty fix for now is to install and login from the real php server ip. By default it's `http://localhost:35410` - the port may change and are specified in the warning like the one you post

When you are logged in the panel, the dev server seems to work correctly so you can use the correct devserver url (default `http://localhost:8080`)

I'll continue to investigate on this

**Comment by danoszz** [comment]
I updated the full content of www folder with the [starterkit](https://github.com/getkirby/starterkit) and the panel seems to be working 'fine'. Of course not with your custom kirby-webpack workflow 😜

**Comment by pqml** [comment]
Yep, something in Kirby 2.5.9 breaks the dev server, I tested kirby webpack with the 2.5.8 version of `panel` and `kirby` and everything is working fine

**Comment by pqml** [comment]
Found the source of the issue, I opened an issue on Kirby toolkit:
https://github.com/getkirby/toolkit/issues/274

**Comment by danoszz** [comment]
Sweet! Thanks for your quick response ✨

**Comment by pqml** [comment]
Fixed in the [0.7.4 release](https://github.com/brocessing/kirby-webpack/releases/tag/0.7.4)

You can also update yours by changing `config.localhost.php`:
Just change the line `c::set('url', '//' . $_SERVER['HTTP_X_FORWARDED_HOST']);`
to `c::set('url', 'http://' . $_SERVER['HTTP_X_FORWARDED_HOST']);`
